contracts/contracts/NoLossDao_v0.sol

```diff
@@ -338,55 +383,49 @@ contract NoLossDao_v0 is Initializable {
 
   /// @dev Anyone can call this every 2 weeks (more specifically every *iteration interval*) to receive a reward, and increment onto the next iteration of voting
   function distributeFunds() external iterationElapsed {
-    // On a *whatever we decide basis* the funds are distributed to the winning project
-    // E.g. every 2 weeks, the project with the most votes gets the generated interest.
-    // figure our what happens with the interest from the first proposal iteration
-    // Possibly make first iteration an extended one for our launch (for marketing)
-    // console.log(
-    //   'Iteration no: ',
-    //   proposalIteration,
-    //   'Time that this iteration has ended incremented ',
-    //   proposalDeadline
-    // );
+    interestReceivers[0] = msg.sender; // Set the miners address to receive a small reward
+
     if (proposalIteration > 0) {
-      // Only if last winner is not withdrawn (i.e. still in cooldown) make it active again
-      if (
-        state[topProject[proposalIteration.sub(1)]] == ProposalState.Cooldown
-      ) {
-        state[topProject[proposalIteration.sub(1)]] = ProposalState.Active;
+      // distribute interest from previous week.
+      uint256 previousIterationTopProject = topProject[proposalIteration.sub(
+        1
+      )];
+      if (previousIterationTopProject != 0) {
+        // Only if last winner is not withdrawn (i.e. st ill in cooldown) make it active again
+        if (state[previousIterationTopProject] == ProposalState.Cooldown) {
```

Comment:
As long as the cool down period only ever gets bigger in the future, this function will still work fine. But we can never decrease the cooldown period.

That said in future versions with a longer cooldown this won't be called `previousIterationTopProject`.